### DISCUSSION TITLE
Project Configuration File

### DISCUSSION THREAD

**Discussion by invictoring** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#12] [General]
Having issues while setting up project configurations like accecss token.  got the token and all other credentials but still message is not delivered. Kindly guide thanks.

**Comment by gabrieldwight** [comment]
Hi @invictoring If it's your first time sending a message with a test phone number. You need to send the default template message from meta developer website and reply with any message text to the test phone number. After that process is done, you can now send messages to your phone number.

**Reply by invictoring** [reply]
Ok thanks alot.
